README.rst

```diff
@@ -18,45 +18,62 @@ Build and Tests
 Build
 ----------------------------------------------------------------------
 
-Ensure you have cmake 2.8 or later version.
+Prerequisite
+^^^^^^^^^^^^^
+ - Ensure you have cmake 2.8 or later version.
+ - For Windows, require: one of Visual Studio [VS14, VS15, VS16, VS17]
+ - For Linux, require: gcc, g++
 
 Linux and OSX
 ^^^^^^^^^^^^^
 
 .. code-block:: bash
 
+    ./scripts/buildDependencies.sh
     ./scripts/build_libsnowflakeclient.sh
 
 Windows
-^^^^^^^^^^
+^^^^^^^
 
 Set environment variables: PLATFORM: [x64, x86], BUILD_TYPE: [Debug, Release], VS_VERSION: [VS14, VS15, VS16, VS17] and run the script.
 
 .. code-block:: bash
-
     set platform=x64
     set build_type=Debug
-    set vs_version=VS14
-    ci\build.bat
+    set vs_version=VS17
+    ci\\build_win.bat
```

Comment:
If possible I would prefer to use similar script for both windows and non-windows. What's the difference between scripts ones and ci ones? which is better for customer to use?